app/assets/javascripts/text-area-autosize.js

```diff
@@ -0,0 +1,3 @@
+$(function() {
+  $(".js-text-area-auto-size").textareaAutoSize();
```

Comment:
I'm not sure of the benefit of rewriting it as a module @fofr.

This is an external library which is tested here https://github.com/javierjulio/textarea-autosize/blob/master/test/test.js